api/src/test/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImplTest.java

```diff
@@ -27,9 +27,12 @@
 import java.text.ParseException;
```

Comment:
Is it possible to add a test/tests here that searches by multiple parameters at the same time?